### DISCUSSION TITLE
Is there any plan for kubernetes integration

### DISCUSSION THREAD

**Discussion by rjshrjndrn** [outline/outline#1333] [General]
Hi,
I was going through the Readme and found that I can spin the entire setup on docker-compose. So was amazing why we don't have a helm chart which can be used in our own cluster? can be used as prod as well. Thought I can contribute a helm chart and was creating one, with all dependencies pre-configured. But in the end I figured out that there is no official Docker image from Outline. So was worrying is there any plans to support Kube and provide an official image so that starters can take it and use it directly?

**Comment by tommoor** [comment]
Hi, there is an official docker image here: https://hub.docker.com/r/outlinewiki/outline

In terms of supporting Kube I'm generally open to it but there isn't anyone on the core team with knowledge of the technology to maintain the helm chart so I'd be worried about it getting out of sync with actual deployment practices.
